backend/db/analysis.py

```diff
@@ -83,18 +104,35 @@ def counts_update(schema: str = SCHEMA, local=False):
             })
 
 
+# TODO: support multiple schema
 def counts_over_time(
     schema: str = SCHEMA, local=False, method=COUNTS_OVER_TIME_OPTIONS[0]
 ):
     """Checks counts of database and store what the results look like in a database over time"""
+    if method not in COUNTS_OVER_TIME_OPTIONS:
+        raise ValueError(f'counts_over_time(): Invalid method {method}. Must be one of {COUNTS_OVER_TIME_OPTIONS}')
     with get_db_connection(schema='', local=local) as con:
+        # Get counts
         counts: List[Dict] = [dict(x) for x in run_sql(con, f'SELECT * from counts;')]
         counts_df = pd.DataFrame(counts)
         counts_df = counts_df[counts_df['schema'] == schema]
         values = 'count' if method == 'print_counts_table' else 'delta'
-        df = counts_df.pivot(index='table', columns='timestamp', values=values)
-        if method == 'save_delta_viz':  # TODO
-            pass
+        data_df = counts_df.pivot(index='table', columns='timestamp', values=values).fillna(0).astype(int)
+        # Add note
+        # todo
+        runs = [dict(x) for x in run_sql(con, f'SELECT timestamp, note from counts_runs;')]
+        timestamps = [x['timestamp'] for x in runs]
+        ts_dict = {}
+        for ts in timestamps:
+            for run in runs:
+                if run['timestamp'] == ts:
+                    ts_dict[ts] = run['note']
+        runs_df = pd.DataFrame([ts_dict])
+        runs_df.index = ['COMMENT']
```

Comment:
@Sigfried Spacing could be better, but heres the initial addition of what comments look like when comparing counts over time:

**Counts**
```tsv
                                                                  2023-04-27 21:10:47.893920 2023-04-18 19:42:08.446779
COMMENT                                    Testing new feature of adding note annotations...                  Baseline.
all_csets                                                                               6314                       6314
code_sets                                                                               6315                       6314
concept                                                                              6902295                    6902295
concept_ancestor                                                                    19524628                   19524628
concept_relationship                                                                17361726                   17361726
concept_relationship_plus                                                           17361726                   17361726
concept_set_container                                                                   4189                       4188
concept_set_counts_clamped                                                              5084                       5084
concept_set_json                                                                           1                          1
concept_set_members                                                                  8274707                    8274707
concept_set_version_item                                                             1824569                    1824569
concepts_with_counts                                                                 6902295                    6902295
cset_members_items                                                                   8524919                    8524919
deidentified_term_usage_by_domain_clamped                                             183386                     183386
omopconceptset                                                                          5155                       5155
omopconceptsetcontainer                                                                 3968                       3968
researcher                                                                              1000                       1000
rxnorm_med_cset                                                                           34                         34
small_snomed                                                                          197214                          0
```

**Deltas**
```tsv
                                                                  2023-04-27 21:10:47.893920 2023-04-18 19:42:08.446779
COMMENT                                    Testing new feature of adding note annotations...                  Baseline.
all_csets                                                                                  0                       6314
code_sets                                                                                  1                       6314
concept                                                                                    0                    6902295
concept_ancestor                                                                           0                   19524628
concept_relationship                                                                       0                   17361726
concept_relationship_plus                                                                  0                   17361726
concept_set_container                                                                      1                       4188
concept_set_counts_clamped                                                                 0                       5084
concept_set_json                                                                           0                          1
concept_set_members                                                                        0                    8274707
concept_set_version_item                                                                   0                    1824569
concepts_with_counts                                                                       0                    6902295
cset_members_items                                                                         0                    8524919
deidentified_term_usage_by_domain_clamped                                                  0                     183386
omopconceptset                                                                             0                       5155
omopconceptsetcontainer                                                                    0                       3968
researcher                                                                                 0                       1000
rxnorm_med_cset                                                                            0                         34
small_snomed                                                                          197214                          0
```